### PR TITLE
refactor(experimental): graphql: add more type aliases

### PR DESCRIPTION
**Pull request by buffalojoec**
Just renaming some types and adding aliases where it makes sense.

### CODE REVIEW

**Comment by buffalojoec** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2095?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `master`
  * **PR #2070** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2070?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #2095** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2095?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
      * **PR #2096** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2096?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #2068** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2068?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #2073** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2073?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #2069** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2069?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2095?utm_source=stack-comment).

File: packages/rpc-graphql/src/resolvers/types.ts [hunk 1]
```diff
@@ -1,12 +1,62 @@
...
+        }
+        return null;
+    },
+    __parseValue(value: string): bigint {
+        return BigInt(value);
+    },
+    __serialize(value: string): bigint {
+        return BigInt(value);
+    },
+};
```
Comment (steveluscher) [inline packages/rpc-graphql/src/resolvers/types.ts:33]: Added via Giphy

File: packages/rpc-graphql/src/__tests__/transaction-tests.ts [hunk 2]
```diff
@@ -168,25 +168,25 @@ describe('transaction', () => {
...
                             }
-                            signatures
                         }
                     }
                 }
-            }
-        `;
-            const result = await rpcGraphQL.query(source);
+            `;
+            const result = await rpcGraphQL.query(source, { signature });
```
Comment (steveluscher) [inline packages/rpc-graphql/src/__tests__/transaction-tests.ts:189]: <3

File: packages/rpc-graphql/src/schema/types.ts [hunk 3]
```diff
@@ -32,6 +54,10 @@ export const commonTypeDefs = /* GraphQL */ `
         rewardType: String
     }
 
+    scalar Signature
+
+    scalar Slot
```
Comment (steveluscher) [inline packages/rpc-graphql/src/schema/types.ts:59]: <i class='graphite__hidden'>[Re: lines 9 to 59]<br/><br/></i>



<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2095?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by steveluscher — APPROVED** [review]
[beautiful](https://app.graphite.dev/macros/solana-labs)

**Comment by buffalojoec** [comment]
### Merge activity

* **Feb 22, 3:57 PM EST**: @buffalojoec started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2095).

**Pull request merged**